Apps/Playground/0.70/metro.config.js

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
Don't we need all our customizations to this file, like support for sym linking?

Do we have all the steps for adding a new RN version documented? Seems like it would be good to have a step by step so we can be sure we got everything when we add a new version.

Reply:
It looks like this file still hasn't been updated. Have you run the 0.70 Playground app locally, because it looks like it shouldn't work.